Tasks/ServiceFabricDeployV1/task.json

```diff
@@ -95,9 +95,9 @@
             "name": "useDiffPackage",
```

Comment:
Can you please show me how the UI looks like?

Reply:
Done